py/build.gradle

```diff
@@ -236,102 +239,108 @@ else:
+        rename ".*", "sparkling_water_assembly.jar"
     }
-    // Configure proper name
-    archiveBaseName = "h2o_pysparkling_${sparkMajorVersion}"
+    // Save comment into module file
+    file("${project.pkgDir}/sparkling_water/").mkdir()
+    file("${project.pkgDir}/sparkling_water/__init__.py").write("# Sparkling-water JAR holder for pySparkling module.")
+}
+// Configure proper name
+archiveBaseName = "h2o_pysparkling_${sparkMajorVersion}"
 
-    from pkgDir
-    destinationDirectory = distDir
+from pkgDir
+destinationDirectory = distDir
 }
 
 configurations {
-    sdist
+sdist
 }
 
 artifacts {
-    sdist distPython
+sdist distPython
 }
 
 def createUnitTestArgs() {
-    return [
-            "${distPython.archiveFile.get()}",
-            "spark.ext.h2o.backend.cluster.mode=${detectBackendClusterMode()}",
-            "spark.ext.h2o.external.disable.version.check=true"
-    ]
+return [
+    "${distPython.archiveFile.get()}",
+    "spark.ext.h2o.backend.cluster.mode=${detectBackendClusterMode()}",
+    "spark.ext.h2o.external.disable.version.check=true"
+]
 }
 
 def createIntegTestArgs() {
-    return [
-            "${distPython.archiveFile.get()}",
-            "spark.testing=true",
-            "spark.ext.h2o.backend.cluster.mode=${detectBackendClusterMode()}",
-            "spark.test.home=${sparkHome}",
-            "spark.ext.h2o.node.log.dir=${buildDir}/h2ologs-itest",
-            "spark.ext.h2o.external.disable.version.check=true"
-    ]
+return [
+    "${distPython.archiveFile.get()}",
+    "spark.testing=true",
+    "spark.ext.h2o.backend.cluster.mode=${detectBackendClusterMode()}",
+    "spark.test.home=${sparkHome}",
+    "spark.ext.h2o.node.log.dir=${buildDir}/h2ologs-itest",
+    "spark.ext.h2o.external.disable.version.check=true"
+]
 }
 
 task testPythonMojoPipeline(type: PythonTask, dependsOn: distPython) {
-    extraArgs(*createUnitTestArgs())
-    command = "tests/test_runner.py tests/unit/with_runtime_spark/test_mojo_pipeline.py"
+extraArgs(*createUnitTestArgs())
+command = "tests/test_runner.py tests/unit/with_runtime_spark/test_mojo_pipeline.py"
 }
 
 task testPython(type: PythonTask, dependsOn: distPython) {
-    extraArgs(*createUnitTestArgs())
-    command = "tests/test_runner.py 'tests/unit --ignore=tests/unit/with_runtime_spark/test_mojo_pipeline.py'"
+extraArgs(*createUnitTestArgs())
+command = "tests/test_runner.py 'tests/unit --ignore=tests/unit/with_runtime_spark/test_mojo_pipeline.py'"
 }
 
 if (project.property("testMojoPipeline") == "true") {
-    testPython.dependsOn testPythonMojoPipeline
+testPython.dependsOn testPythonMojoPipeline
 }
 
-task integTestPython(type: PythonTask, dependsOn: [distPython, checkSparkVersionTask]) {
-    extraArgs(*createIntegTestArgs())
-    command = "tests/test_runner.py 'tests/integration --ignore=tests/integration/external_only'"
+task integTestPython(type: PythonTask, dependsOn: [
+distPython,
+checkSparkVersionTask
+]) {
+extraArgs(*createIntegTestArgs())
```

Comment:
indentation of task body would be also great.

Reply:
Yup, was broken in `py`